packages/sdk-gated/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "@lens-protocol/sdk-gated",
+  "version": "1.2.0",
+  "private": true,
+  "description": "SDK for encrypting and decrypting gated content on Lens Protocol",
+  "author": "zannis@aave.com",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "browser": "./dist/browser.js",
+  "types": "./dist/index.d.ts",
+  "license": "MIT",
+  "files": [
+    "dist",
+    "package.json",
+    "CHANGELOG.md"
+  ],
+  "scripts": {
+    "build": "pnpm clean && pnpm rollup:prod",
```

Comment:
`rollup:prod` script is running `clean` already so here it is run twice

Reply:
Good catch, removed